lib/v8debugapi.js

```diff
@@ -277,6 +280,12 @@ module.exports.create = function(logger_, config_) {
     // TODO: need test coverage for the above
     // TODO: work with debug server to come up with a more robust solution
 
+    if (breakpoint.location.line >= matchingScript.line_ends.length) {
+      return setErrorStatusAndCallback(cb, breakpoint,
+        StatusMessage.BREAKPOINT_SOURCE_LOCATION,
+        messages.INVALID_LINE_NUMBER);
```

Comment:
It is consistent with the other occurrences of setErrorStatusAndCallback in the file. Also, we need to pick a style-guide to refer to for this; and somehow enforce as well. What style guide should we use?